test/integration/inviting_users_test.rb

```diff
@@ -11,7 +11,24 @@ class InvitingUsersTest < ActionDispatch::IntegrationTest
     fill_in "Confirm new password", with: "this 1s 4 v3333ry s3cur3 p4ssw0rd.!Z"
     click_button "Save password"
 
-    assert_response_contains("You are now signed in")
+    assert_response_contains("Your password was set successfully.")
+  end
+
+  should "require the invited user to sign in after setting their password" do
+    user = User.invite!(name: "Neptuno Keighley", email: "neptuno.keighley@office.gov.uk")
+
+    accept_invitation(
+      invitation_token: user.raw_invitation_token,
+      password: "pretext annoying headpiece waviness header slinky",
+    )
+
+    assert_response_contains("Sign in to GOV.UK")
```

Comment:
Does this page make it clear that you now need to sign in? Perhaps it's obvious. I mainly want to make sure we're not going to introduce a whole new class of support tickets.

Reply:
I'll ask the team for a second opinion before I merge it.

But here's the page you land on from the invite link:

![Screenshot from 2023-08-30 10-09-27](https://github.com/alphagov/signon/assets/141013432/bc644da6-ab27-44c2-b104-cc98cf230d9d)

And here's the next page:

![Screenshot from 2023-08-30 10-10-03](https://github.com/alphagov/signon/assets/141013432/aee54ffd-d00e-400a-8487-5c8cef2cac14)

Reply:
Ah, nice! I reckon it's fine, but probably worth a 2nd opinion.